### DISCUSSION TITLE
Question on voxel resolution

### DISCUSSION THREAD

**Discussion by tomyamashita** [r-lidar/lidR#480] [Q&A] [answered]
When you compute voxel metrics and set the resolution, I am curious as to what the bottom voxel represents? The function outputs Z values but are these the top, bottom, or center of the voxel? 
For example, if we were computing the number of points in each voxel using `stdmetrics_ctrl`: 

```r
voxel.metrics <- voxel_metrics(las, ~.stdmetrics_ctrl, res = c(10,2))
```

and we got a series of Z values for a pixel that were 5, 7, and 9, what do those represent? For Z=5, does this represent points between 5 and 7, 3 and 5, or 4 and 6?

**Comment by Jean-Romain** [comment, marked as answer]
Point returned are voxels centers. So z = 5 with a resolution of 2 is Z in [4, 6[. Or ]4, 6] I don't know, but I don't think it matters much.

**Reply by tomyamashita** [reply]
Thanks. We are looking at how shrub density varies at different heights above the ground so this is useful for us to know.
